components/Changers/FontSizeChanger.jsx

```diff
@@ -0,0 +1,25 @@
+import styles from "@/styles/components/changers/font-size-changer.module.scss";
+
+export default function FontSizeChanger({ fontSize, setFontSize }) {
```

Comment:
Don't remember if I commented on it elsewhere but a better name would be `CounterInput` or something

Reply:
noted